client/app/components/Header/Header.scss

```diff
@@ -97,6 +97,12 @@
   }
 
   &Desktop {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1000;
```

Comment:
It would be a better design to have this background only show up once the user starts scrolling in the page. We can do that by applying the transparent black background through React JS in [/client/app/components/Header/index.jsx](https://github.com/ifmeorg/ifme/blob/main/client/app/components/Header/index.jsx)